apps/web/src/actions/commits/fetchCommitsByProjectAction.test.ts

```diff
@@ -0,0 +1,83 @@
+import { CommitStatus, factories } from '@latitude-data/core'
+import type { Workspace } from '@latitude-data/core/browser'
+import useTestDatabase from '$core/tests/useTestDatabase'
+import { beforeEach, describe, expect, it, vi } from 'vitest'
+
+import { fetchCommitsByProjectAction } from './fetchCommitsByProjectAction'
+
+const mocks = vi.hoisted(() => {
+  return {
+    getSession: vi.fn(),
+  }
+})
+vi.mock('$/services/auth/getSession', () => ({
+  getSession: mocks.getSession,
+}))
+
+let workspace: Workspace
+describe('getUsersAction', () => {
+  useTestDatabase()
```

Comment:
you don't need it

Reply:
Ok but I keep watching data in test db after test run. We have something wrong. Doesn't matter if I put this here or not